### PR TITLE
Finish task design UI calendar index

### PR DESCRIPTION
**Pull request by tqtrung09**

### CODE REVIEW

**Comment by nguyenthaison** [comment]
Oke

File: app/assets/stylesheets/calendars.css.scss [hunk 1]
```diff
@@ -1,18 +1,108 @@
...
+    cursor: pointer;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    background-image: none;
+    border: 1px solid transparent;
+    border-radius: 4px;
+    text-decoration: none;
+    font-family: sans-serif;
```
Comment (nhunght) [inline app/assets/stylesheets/calendars.css.scss]: move to custom.css.scss

**Comment by daodat18plus** [comment]
OK

**Comment by daodat18plus** [comment]
OK

**Comment by phornsopheak** [comment]
ok

**Pull request merged**